MAVProxy/modules/mavproxy_map/srtm.py

```diff
@@ -75,7 +74,7 @@ def __init__(self, server="firmware.ardupilot.org",
             try:
                 cachedir = os.path.join(os.environ['HOME'], '.tilecache/SRTM')
             except Exception:
-                cachedir = os.path.join(tempfile.gettempdir(), 'MAVProxySRTM')
+                cachedir = os.path.join(os.environ['LOCALAPPDATA'], '.tilecache/SRTM')
```

Comment:
I think we need to check if 'LOCALAPPDATA' exists in os.environ, otherwise we'll get another nested exception

Reply:
Fixed